config/sample/generis.conf.php

```diff
@@ -43,7 +43,7 @@
 define('ROOT_URL', '');
 
 # language
-define('DEFAULT_LANG', '');
+define('DEFAULT_LANG', 'en-US');
```

Comment:
I am wondering this was empty for some other reason. Sure it will not break some logic in other place?

Reply:
@bartlomiejmarszal why you've changed this one, but left `DEFAULT_ANONYMOUS_INTERFACE_LANG` with empty string  ?

Reply:
This change was made here: 

https://github.com/oat-sa/generis/pull/834/commits/168add5780c37c8969a7d387ee3e974cd52e43b4

@gitromba ?

Reply:
It would be quite a surprise if this would be breaking something as the file is in the `sample` folder. There should be no factory code endangered IMO.

I can think of although to break some tests (and maybe this is what @gabrielfs7 is thinking of too). This is unlikely though as this is a fix for `DEFAULT_LANG`. It was defined, but with an empty string.

Reply:
@boajer I have no clue what is `DEFAULT_ANONYMOUS_INTERFACE_LANG` for. I've changed `DEFAULT_LANG` because otherwise `UserLanguageServiceTest` was broken.

Reply:
I was not sure about this and why we defined mostly empty, but now we set some values. It seems the file is not a sample anymore. Is there is not risk, we can keep. Anyway this change should not be part of this PR since it was done in a different PR.

Reply:
I will resolve this. If you feel we need to discuss more, tell me :) 

Thanks for the clarification @bartlomiejmarszal @gitromba